securedrop/journalist_templates/admin.html

```diff
@@ -85,7 +85,7 @@ <h2 id="users-heading" class="visually-hidden">Users</h2>
 
 <a href="{{ url_for('admin.manage_config') }}" class="btn icon icon-edit section-spacing-inline"
   id="update-instance-config" aria-label="{{ gettext('Update instance configuration') }}">
-  {{ gettext('INSTANCE CONFIG') }}
+  {{ gettext('INSTANCE CONFIGURATION') }}
```

Comment:
This is probably out of scope for WCAG annotations and will require translation changes.

Reply:
okay, sounds good - will remove for now